tests/v2/validation/snapshot/snapshot.go

```diff
@@ -208,18 +208,21 @@ func snapshotRKE1(t *testing.T, client *rancher.Client, podTemplate corev1.PodTe
 		}
 	}
 
-	return initialSnapshotConfig{initialKubernetesVersion, initialControlPlaneUnavailable, initialWorkerUnavailableValue, snapshotToRestore}
+	return cluster, initialSnapshotConfig{initialKubernetesVersion, initialControlPlaneUnavailable, initialWorkerUnavailableValue, snapshotToRestore}
```

Comment:
why is there a need to return the initialSnapshotConfig now that we're returning the entire cluster object?

Reply:
L238 is an example of why this is still here. We want to make sure that the current values match what the value is prior to restoring a snapshot.

This especially is important when we're doing upgrade scenarios.

Reply:
but the cluster object returned here has that information in it, correct? why not just use that instead of using the custom object?

Reply:
Discussed offline, resolved this.